tests/Feature/Lib/RootAlbumUnitTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * We don't care for unhandled exceptions in tests.
+ * It is the nature of a test to throw an exception.
+ * Without this suppression we had 100+ Linter warning in this file which
+ * don't help anything.
+ *
+ * @noinspection PhpDocMissingThrowsInspection
+ * @noinspection PhpUnhandledExceptionInspection
+ */
+
+namespace Tests\Feature\Lib;
+
+use Illuminate\Testing\TestResponse;
+use Tests\TestCase;
+
+class RootAlbumUnitTest
+{
+	private TestCase $testCase;
+
+	public function __construct(TestCase $testCase)
+	{
+		$this->testCase = $testCase;
+	}
+
+	/**
+	 * Gets the root album.
+	 *
+	 * @param int         $expectedStatusCode
+	 * @param string|null $assertSee
+	 * @param string|null $assertDontSee
+	 *
+	 * @return TestResponse
+	 */
+	public function get(
+		int $expectedStatusCode = 200,
+		?string $assertSee = null,
+		?string $assertDontSee = null
+	): TestResponse {
+		$response = $this->testCase->postJson('/api/Albums::get');
+		$response->assertStatus($expectedStatusCode);
+		if ($assertSee) {
```

Comment:
Good thing that phpstan is not parsing those files. :laughing:

Reply:
Yes, I had the exact same though. I also inclined to write `if ($asserSee !== null)`, but this is copy & paste from the other files and then I decided to keep it consistent. At some point in the future I would like to include the tests into PHPStan, but not now.